src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/custom.py

```diff
@@ -1,155 +0,0 @@
-# --------------------------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why is this file being removed? I suspect this was unintentional.

Reply:
If you revert this change you'll also need to revert the change to the configure module versions since there won't be any differences.